apis/apps/v1alpha1/containerrecreaterequest_types.go

```diff
@@ -172,6 +177,16 @@ type ContainerRecreateRequestSyncContainerStatus struct {
 	ContainerID string `json:"containerID,omitempty"`
 }
 
+// ContainerRecreateRequestKilledContainerStatus contains the state of the last killed container.
+type ContainerRecreateRequestForceKilledContainerStatus struct {
+	// Name of the container.
+	Name string `json:"name"`
+	// Container is killed by force
+	IsForceKilled bool `json:"restartCount"`
```

Comment:
struct tag for IsForceKilled should be isForceKilled or something similar. restartCount is considered as something different.

Reply:
deleted